packages/drivers/redux-form/package.json

```diff
@@ -10,6 +10,8 @@
   "devDependencies": {
     "cross-env": "^5.2.0",
     "k-ramel": "^1.2.0",
+    "react-redux": "^5.1.1",
```

Comment:
why ?

Reply:
Because it is needed, we had a warning before and now it failed tests because no one installed it (since redux-little-router is removed)

Reply:
(it worked before because of hoisting)